binder/form.go

```diff
@@ -66,7 +66,27 @@ func (b *FormBinding) bindMultipart(req *fasthttp.Request, out any) error {
 		return err
 	}
 
-	return parse(b.Name(), out, data.Value)
+	temp := make(map[string][]string)
+	for key, values := range data.Value {
```

Comment:
this part looks almost identical to the other square bracket parsings, if it were possible it would be cool if we could move this into another function and use it in all three places

query params
https://github.com/gofiber/fiber/blob/775e0a73f3fc0eba0940488e2accd3b87ab245ee/binder/query.go#L34-L45
form params urlencoded
https://github.com/gofiber/fiber/blob/775e0a73f3fc0eba0940488e2accd3b87ab245ee/binder/form.go#L41-L52